Formula/xcbuild.rb

```diff
@@ -0,0 +1,21 @@
+class Xcbuild < Formula
+  desc "Xcode-compatible build tool"
+  homepage "https://github.com/facebook/xcbuild"
+  url "https://github.com/facebook/xcbuild.git", :tag => "0.1.0", :revision => "08575f65f1c907e280ed4663b5461b7cfaf5cfaa"
```

Comment:
Or better yet, use [the release](https://github.com/facebook/xcbuild/releases/tag/0.1.0) instead. We use tags only when releases aren't available.